google/cloud/forseti/services/inventory/base/resources.py

```diff
@@ -517,6 +517,47 @@ def key(self):
     return ResourceSubclass
 
 
+def k8_resource_class_factory(resource_type, hash_key=False):
+    """Factory function to generate Kubernetes Resource subclasses.
+
+    Args:
+        resource_type (str): The static Kubernetes resource type for this
+        subclass.
+        hash_key (bool): If true, use a hash of the key field data instead of
```

Comment:
Do you ever expect to set hash_key to False? If not, I recommend removing it from the arguments and just always returning the hash of the uid.

Reply:
Done